### ISSUE TITLE
Fix failing v2-beta terminfo test

### ISSUE THREAD

**Issue by johnbradley** [phenoscape/rphenoscape#210]
A terminfo test is failing when using the [v2-beta kb.phenoscape API](https://dev.phenoscape.org/api/v2-beta)
```
Error (test-pk.R:323:3): creating terminfo objects and extracting properties
Error: 'list' object cannot be coerced to type 'double'
```
https://github.com/phenoscape/rphenoscape/blob/82e16d96e3ed34c6ddd59ee640ce58310d684c62/tests/testthat/test-pk.R#L323
This line is failing because some of the terms returned have no synonyms.
```
> terms = find_term("basihyal bone")
> objs <- as.terminfo(terms$id)
> objs[[4]]
terminfo 'basihyal bone aplastic, abnormal' http://purl.obolibrary.org/obo/ZP_0002752
Definition: Abnormal(ly) aplastic (of) basihyal bone.
> objs[[4]]$synonyms
named list()
```

**Comment by hlapp** [comment]
Looks like the test is too strict. We don't need to have _every_ term to have synonyms to declare success, only, say, half, or some relatively conservative fraction (so that failures alert to a real problem rather than just some changes in the data).

**Comment by johnbradley** [comment]
Only 4 of the 14 terms have synonyms so ~ 28% have synonyms:
```
                                             ids                                         labels has_synonyms
1  http://purl.obolibrary.org/obo/UBERON_0011618                                  basihyal bone         TRUE
2      http://purl.obolibrary.org/obo/MP_0009917            abnormal hyoid bone body morphology         TRUE
3      http://purl.obolibrary.org/obo/MP_0030162                         absent hyoid bone body         TRUE
4      http://purl.obolibrary.org/obo/ZP_0002752               basihyal bone aplastic, abnormal        FALSE
5      http://purl.obolibrary.org/obo/ZP_0002821               basihyal bone ossified, abnormal        FALSE
6      http://purl.obolibrary.org/obo/ZP_0014146                basihyal bone present, abnormal        FALSE
7      http://purl.obolibrary.org/obo/ZP_0016130                   basihyal bone bent, abnormal        FALSE
8      http://purl.obolibrary.org/obo/ZP_0022076             basihyal bone morphology, abnormal        FALSE
9      http://purl.obolibrary.org/obo/ZP_0017745               basihyal bone position, abnormal        FALSE
10     http://purl.obolibrary.org/obo/ZP_0109835                basihyal bone quality, abnormal        FALSE
11    http://purl.obolibrary.org/obo/ZFA_0000316                      basihyal bone (zebrafish)         TRUE
12     http://purl.obolibrary.org/obo/ZP_0002710       basihyal bone decreased length, abnormal        FALSE
13     http://purl.obolibrary.org/obo/ZP_0005833         basihyal bone decreased size, abnormal        FALSE
14     http://purl.obolibrary.org/obo/ZP_0002711 basihyal bone mislocalised ventrally, abnormal        FALSE
```

**Comment by balhoff** [comment]
All these ZP terms are new in the new pipeline; not found in the old KB.

**Comment by hlapp** [comment]
@johnbradley perhaps best to filter out ZP?

**Comment by johnbradley** [comment]
Fixed by PR #219
